crypto/s390xcap.c

```diff
@@ -86,16 +86,20 @@ void OPENSSL_cpuid_setup(void)
     {
```

Comment:
change
```
#ifdef OSSL_IMPLEMENT_GETAUXVAL
```
to
```
#if defined(OSSL_IMPLEMENT_GETAUXVAL) && defined(HWCAP_STFLE) && defined(HWCAP_S390_VX)
```
so that in case any of the defines is missing, the old signal handler stuff is used.

With your proposed changes, systems that define OSSL_IMPLEMENT_GETAUXVAL ..
- .. but dont define HWCAP_S390_STFLE would disable all HW support.
- .. but dont define HWCAP_S390_VX would run into SIGFPE in case machine has vectory facility but kernel disables it.

Reply:
Ah, ok.  Sure

Reply:
Fixed